test/integration/features/cmd-docker-env.feature

```diff
@@ -0,0 +1,41 @@
+@cmd-docker-env @command
+Feature: Command docker-env
+Command docker-env helps the user to set environmental variables which are needed force
+the usage of Minishift with docker command on VM host.
+INFO: This feature runs against persistent shell instance. To use non-default shell, please
```

Comment:
"persistent shell instance" - persistent?

Reply:
@hferentschik Wanted to highlight the fact that environmental variables and other modifications done will persist until the moment the shell instance is closed, but sure it can be deleted, wdyt?

Reply:
@agajdosi use of `persistent shell instance` is confusing to me too.

Reply:
ok, will remove, thanks 👍